src/openrct2/world/TileElement.cpp

```diff
@@ -242,5 +264,10 @@ int32_t TileElementBase::GetClearanceZ() const
 
 void TileElementBase::SetClearanceZ(int32_t newZ)
 {
-    clearance_height = (newZ / COORDS_Z_STEP);
+    uint8_t new_clearance_height = (newZ / COORDS_Z_STEP);
+    if (clearance_height != new_clearance_height)
+    {
+        mapScheduleCalcHighestTileHeight();
```

Comment:
If the clearance height of an element is changed then a highest tile calc is scheduled.